bayesian.go

```diff
@@ -273,6 +375,10 @@ func (c *Classifier) Learn(document []string, which Class) {
 // Unlike c.Probabilities(), this function is not prone to
 // floating point underflow and is relatively safe to use.
 func (c *Classifier) LogScores(document []string) (scores []float64, inx int, strict bool) {
+	if c.tfIdf && !c.didConvertTfIdf {
+		panic("Error:Using TD-IDF classifier:Need to call ConvertTermsFreqsToTFIDF's first ")
```

Comment:
Change error message to "Using a TF-IDF classifier. Please call ConvertTermsFreqToTfIdf before calling LogScores."